extensions/podman/packages/extension/src/podman-install.ts

```diff
@@ -674,10 +668,7 @@ export class VirtualMachinePlatformCheck extends BaseCheck {
 abstract class WindowsCheck extends BaseCheck {
   async isUserAdmin(): Promise<boolean> {
     try {
-      const { stdout: res } = await extensionApi.process.exec('powershell.exe', [
-        '$null -ne (whoami /groups /fo csv | ConvertFrom-Csv | Where-Object {$_.SID -eq "S-1-5-32-544"})',
-      ]);
-      return res.trim() === 'True';
+      return await (await getPowerShellClient()).isUserAdmin();
```

Comment:
I think we reached the maximum of await instructions
we can remove the first one

Reply:
Fixed